doc/services.rst

```diff
@@ -47,6 +49,10 @@ Amazon EventBridge
 -----------
 .. autoclass:: stepfunctions.steps.service.EventBridgePutEventsStep
 
+Amazon Glue DataBrew
+--------------------
+.. autoclass:: stepfunctions.steps.service.DataBrewStartJobRunStep
```

Comment:
should this name also include `Glue` in the name? It doesn't have to map to the name in the service principal (`databrew`)

Reply:
Agreed! I was debating adding it as well - will make the changes in the next commit